zerver/lib/bugdown/__init__.py

```diff
@@ -1173,6 +1174,15 @@ def run(self, lines):
                 if re.search(match_re, content):
                     current_message.alert_words.add(word)
 
+            for command in slash_commands:
+                escaped = re.escape(("/" + command).lower())
+                match_re = re.compile(u'(?:%s)%s(?:%s)' %
+                                      (allowed_before_punctuation,
+                                       escaped,
+                                       allowed_after_punctuation))
+                if re.search(match_re, content):
+                    current_message.slash_commands.add(command)
```

Comment:
Just wondering, since you're using a `set` here, do you ever expect a situation where a message triggers more than one slash command?